

For now you can still switch to reload with -mno-lra.

Successful bootstrap with c,c++ natively on m68k-linux-gnu.
Test suite results on par with previous non-LRA bootstrap.
A previous version also bootstrapped Ada.

Is this Ok for trunk?

gcc/
	PR target/113939
	* config/m68k/m68k.opt (mlra): Default to LRA instead of reload.
---
 gcc/config/m68k/m68k.opt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/gcc/config/m68k/m68k.opt b/gcc/config/m68k/m68k.opt
index a97fac654d8..e9693ff681b 100644
--- a/gcc/config/m68k/m68k.opt
+++ b/gcc/config/m68k/m68k.opt
@@ -147,7 +147,7 @@ Target RejectNegative Mask(LONG_JUMP_TABLE_OFFSETS)
 Use 32-bit offsets in jump tables rather than 16-bit offsets.
 
 mlra
-Target Var(m68k_lra_p) Undocumented
+Target Var(m68k_lra_p) Init(1) Undocumented
 Use LRA for reload instead of the old reload framework.  This option is
 experimental, and it may be removed in future versions of the compiler.
 
-- 
2.52.0


